resources/public/api/conf/1.0/common/queryParameters.yaml

```diff
@@ -0,0 +1,57 @@
+components:
+  parameters:
+    taxYear:
+      name: taxYear
+      in: path
+      description: |
+        The tax year the data applies to. If not supplied it will default to the last completed tax year. The earliest permitted tax year is 2017-18.
+      schema:
+        type: string
+        example: 2022-23
+      required: false
+    typeOfBusiness:
+      name: typeOfBusiness
+      in: path
+      description: |
+        The type of business whose obligations are to be returned, must be one of: <ul><li>`self-employment`</li><li>`uk-property`</li><li>`foreign-property`</li></ul>If the type is not specified the default is to return obligations for all businesses. The type must be provided if "businessId" is provided.
```

Comment:
The allowed types can be defined in OAS as an "enum". Then it doesn't need (shouldn't really have) the inline HTML bullets listing the allowed values